### PR TITLE
Port the scoring rubric to a second category (finetuned_chat)

### PR DESCRIPTION
**Pull request by ccerv1**
The base-model rubric was reverse-engineered from one category's scores. This tests whether it was a general mechanism or a bespoke one.

## The result

**32 of 46 products reproduced against the pilot rubric with nothing changed** — no aliases, no `reads` declarations, no corrections. That is the number worth quoting, and it is recorded in the recipe, because the headline 43/43 comes after correcting three scores and would be circular.

The tier ladder and the formula port intact. What needed adding was vocabulary, not structure. `base_pretrained` holds at 47/47 and 195 evidence rows throughout, which was the constraint the change had to respect.

## Scores this changes

Three published scores move. Each was surfaced by the port, none by re-research.

| product | was | now | why |
|---|---|---|---|
| `codegemma` | 3 open_weights | **2 restricted** | Identical facts and the same license family as `gemma-3`, which scores 2. Its own note already read "use-restricted". |
| `tulu-3` | 3 restricted | **2 restricted** | `3/restricted` is a pair the ladder cannot emit. The extra point credited AI2's open post-training layer on the checkpoint. |
| `tinyllama-1-1b-chat-v1-0` | 4 open_weights | **5 open_source** | `zephyr` scores 5 on the same two post-training datasets while sitting on Mistral-7B, whose corpus is closed outright. |

On `tulu-3`: a fine-tuning approach generalizes across bases — the same AI2 recipe produces `olmo-3-instruct` at 5/open_source. Scoring the checkpoint on the method either buries the method or overstates the artifact. The map already resolves this by scoring the method's artifacts as their own products: `tulu-3-sft-mixture` carries 5/open in `training_synthetic_datasets`. What a user downloading the Llama-based checkpoint is bound by is Meta's license, which is 2. That pair of scores is the signal.

## A bug this fixes

The serializer emitted evidence under the raw `components` key, so `post-training-data` would never have matched the formula's `data` condition. Six products would have fallen through to `otherwise` and scored 3 on an absence — no error, no warning. Evidence is now emitted under the dimension name the warehouse joins on, raw keys are kept as traceability rows, and a test asserts one value per grain.

## Machinery added

- **Recorded-name license aliases**, in `signal_routing.yaml` beside the Hub table that solves the same problem for published slugs. Aliases resolve names only — tier assignment stays a per-category judgment, as that file already argued. Left unmapped, a present use-restricting license reads as *absent*, which is the direction that overstates openness.
- **`reads` on a dimension**, naming which recorded keys answer it, preferring the first key whose value is in the enum over the first key present.
- **`deferred`**, for products the rubric does not decide, each with a reason, printed every run and excluded from the reproduction count rather than counted as passing.

## Deferred, not scored

Three products share one cause: a license restricting **conduct** rather than commerce, which the ladder has no tier for. This is not local to this category — the same licenses are scored three ways across the map, including a contradiction where `DeepSeek-Model-License` is `permissive_non_osi` in `base_pretrained` (`deepseek-v3-base`, 3/open_weights) and use-restricting here (2/restricted). Adding a tier on one category's evidence is what #106 exists to prevent, so it goes to an issue.

## Left alone deliberately

- `nemotron-3` and `nemotron-3-nvidia` are the same product entered twice, once per category, both `display_name: 'Nemotron 3'`.
- `granite-code-instruct` records `data` about its base corpus, which this category's vocabulary spends on post-training data. Now warns rather than dropping the value silently.
- A typed `base:` link from a tune to its base model. `lineage` exists but is dataset-only and free-text.

## Test plan

- `uv run python -m build.check_rubric` — `base_pretrained` 47/47, `finetuned_chat` 43/43 with 3 deferred
- `uv run python -m build.validate` — 0 errors
- `uv run python -m build.serialize_rubric --check` — no errors; one new warning, for the `granite` relabel
- `uv run python -m pytest` — 115 passed, up from 92

### CODE REVIEW

**Comment by ccerv1** [comment]
Deployed and verified end to end. Both categories reproduce **through the platform**, checked per product against `sources/scores` rather than by row count:

```
base_pretrained   47/47 reproduce
finetuned_chat    43/43 reproduce, 3 deferred
```

Runs: registry static models `54fb49dd` SUCCESS → `evidence.product_evidence` `6c4b1e77` SUCCESS → `scores.openness_computed` `14ed2d0e` SUCCESS.

### The warehouse needed a matching fix, which is not in this diff

`scores_openness_computed` mirrors `check_rubric.normalize_license` as two `REGEXP_REPLACE` calls, and it never consulted the alias table. So the recorded-name aliases this PR adds resolved locally but not in the warehouse: `check_rubric` reported `base_pretrained` 47/47 while the warehouse abstained on `glm-4-9b` and `nemotron-3-nvidia`, scoring both null on "license maps to no declared tier".

Fixed in **`scores.openness_computed` revision #6**, which adds a `doc_license_aliased` CTE joining `currentai.registry.license_aliases` on `source = 'recorded'` — mirroring what the hub path already did. Noting it here because `udms/` lives outside this repo and is not version controlled, so this comment and the platform's revision history are the only record.

The general point is worth keeping: **any logic hand-mirrored between `build/` and the warehouse SQL can drift, and only an end-to-end check catches it.** A green `check_rubric` is not evidence the platform agrees.

### One hole left open

The three deferred products do come back null from the warehouse — but only because their licenses are undeclared, so the pre-existing unmapped-license guard fires. The recipe's `deferred` block never crosses the bridge. A deferral for any other reason (an ambiguous `data` value, say) would be scored by the warehouse regardless. That wants a `category_deferrals` table and a filter; it is not in this PR.

Also worth flagging while these files are open: `pyoso` served a stale result set immediately after the fix materialized, reporting the two products as still unscored when `execute_sql` showed them correct. Verification through pyoso needs the query text varied.
